grizli/prep.py

```diff
@@ -3499,8 +3647,8 @@ def process_direct_grism_visit(direct={},
             table_to_radec(cat[clip][so], '{0}.cat.radec'.format(direct['product']))
 
         if (fix_stars) & (not isACS) & (not isWFPC2):
-            fix_star_centers(root=direct['product'], drizzle=False, 
-                             mag_lim=19.5)
+            fix_star_centers(root=direct['product'], drizzle=False, isJWST=isJWST, 
```

Comment:
Let's not do this at all for JWST for now

Reply:
Fixed: 5523235dd71bb1b77d7043eae1569cba8b2362cd